src/tape/ensemble.py

```diff
@@ -1847,6 +1847,36 @@ def _sync_tables(self):
         self.object.set_dirty(False)
         return self
 
+    def select_random_timeseries(self, seed=None):
+        """Selects a random lightcurve from the Ensemble
+
+        Parameters
+        ----------
+        seed: int, or None
+            Sets a seed to return the same object id on successive runs. `None`
+            by default, in which case a seed is not set for the operation.
+
+        Returns
+        -------
+        ts: `TimeSeries`
+            Timeseries for a single object
+
+        """
+
+        if seed is not None:
+            np.random.seed(seed)
+
+        # Avoid a choice from full index space, select a random partition to grab from
```

Comment:
Do we know number of objects per partition here? It would be nice to select an integer randomly over all objects, and then get a partition where our integer index is located.

Reply:
We don't and that will be expensive at scale (to grab lengths)

Reply:
Sounds reasonable! Could you please update the docstring to mention that we do **not** sample uniformly here?

Reply:
Done